### PR TITLE
Modal component

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
Feat : Modal comp added.
- sample modal
- modal launch on click

Live preview : https://tender-visvesvaraya-e6aa4a.netlify.app/

### CODE REVIEW

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  flex-wrap: wrap;
+}
+.ct-modal-footer button {
+  margin: 0.25rem;
+}
+
+.ct-modal-action {
+  position: absolute;
+  width: 400px;
+  top: -100px;
```
Comment (amarphule) [inline modal/modal.css:43]: I think you should follow single units either go with `Px` or `rem`.

File: navbar/nav.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@import url("/style.css");
+@import url("/button/button.css");
+@import url("/input/input.css");
+@import url("/badge/badge.css");
+
+header {
+  /* min-height: 100px; */
+  background-color: #eeeeee;
```
Comment (amarphule) [inline navbar/nav.css]: use variable name instead hex color

Comment (MiheerTamkhane) [reply navbar/nav.css]: yup changed I forgot. Thanks buddy

File: modal/modal.css (hunk 1 above)
Comment (MiheerTamkhane) [reply modal/modal.css:43]: yup changed.

**Review by sanathkumar26700 — COMMENTED** [review]
Use only one unit throughout the code preferably rem as it depends on the font size of the root it will be consistent and easy to control.

**Pull request merged**